includes/tutor-general-functions.php

```diff
@@ -1752,4 +1753,55 @@ function tutor_is_local_env() {
 	}
 }
 
+if ( ! function_exists( 'tutor_add_to_cart' ) ) {
+	/**
+	 * Handle add to cart functionalities
+	 *
+	 * @since 3.5.0
+	 *
+	 * @param int $item_id Item id.
+	 *
+	 * @return object {success, message, body}
+	 */
+	function tutor_add_to_cart( int $item_id ) {
+		$monetization      = tutor_utils()->get_option( 'monetize_by' );
+		$response          = new stdClass();
+		$response->success = true;
+		$response->message = __( 'Course added to cart', 'tutor' );
+		$response->body    = null;
+		try {
+			$cart = CartFactory::create_cart( $monetization );
```

Comment:
Instead of returning only url in `body`, we return full cart info in `data` key
```php
$response = [
    'success' => true,
    'message' => __( 'Cart details', 'tutor' ),
    'data' => [
       'cart_url' => $cart->get_cart_url(),
       'items' => $cart->get_cart_items(),
       'total_item' => $cart->get_count(),
       ...
       ....
    ]
]
```